.github/ISSUE_TEMPLATE/2_bug_report_linux.yml

```diff
@@ -0,0 +1,73 @@
+name: "Report a Linux Issue"
```

Comment:
Do we need exact duplicate of the template for each OS? They are all the same, except the label, which in my opinion doesn't help much with categorizing issues.

Reply:
they aren't the same though, there are subtle differences, eg what informations should be provided or instructions how to produce logs.

imo it is one less label that has to be set manually in the best case and i personally find it quite helpful to know which issues i have to take a look at without looking at the content of the issue itself.

Reply:
The issues are very minor, but ok. Maybe out of scope of this pr.